adapters/ix/ix.go

```diff
@@ -1,267 +1,189 @@
 package ix
 
 import (
-	"bytes"
-	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 
-	"github.com/prebid/prebid-server/openrtb_ext"
-
-	"github.com/prebid/prebid-server/pbs"
-
-	"golang.org/x/net/context/ctxhttp"
-
+	"github.com/golang/glog"
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
-// maximum number of bid requests
-const requestLimit = 20
+var maxRequests = 20 // it's not a const for the unit test convenience
 
 type IxAdapter struct {
 	http *adapters.HTTPAdapter
```

Comment:
The `http *adapters.HTTPAdapter` value is not valid for modern bidders. That's a legacy model specific thing. Please remove it.

Reply:
I'll be putting back the legacy interface.

Reply:
Sounds good. You can throw it in legacy.go and legacy_test.go to keep it isolated. I think that's what Conversant did.